src/components/filters/Filters.jsx

```diff
@@ -0,0 +1,211 @@
+            {
+              <input
+                type="checkbox"
+                id="cat4"
+                value="Cleanser"
+                checked={Cleanser}
+                onChange={() => dispatch({ type: "CLEANSER" })}
+              />
+            }
+            <label htmlFor="cat4"> Cleanser</label>
+          </div>
+          <div className="filter-content">
+            {
+              <input
+                type="checkbox"
+                id="cat5"
+                value="Bodycare"
+                checked={Bodycare}
+                onChange={() => dispatch({ type: "BODYCARE" })}
+              />
+            }
+            <label htmlFor="cat5"> Bodycare</label>
+          </div>
+        </div>
+
+        <div className="filter-type-container filter-ratings">
+          <p className="filter-name txt-normal ">Avg. Customer Rating</p>
+          <div className="filter-content">
+            <input
+              type="radio"
+              name="filter-rating-star"
+              id="filter-star-four"
+              value="4"
+              checked={rating === 4}
+              onChange={(e) =>
+                dispatch({ type: "RATING", payload: parseInt(e.target.value) })
+              }
+            />
+            <label htmlFor="filter-star-four">
+              {" "}
+              4 <i className="fas fa-star"></i> & Up
+            </label>
+          </div>
+          <div className="filter-content">
+            <input
+              type="radio"
+              name="filter-rating-star"
+              id="filter-star-three"
+              value="3"
+              checked={rating === 3}
+              onChange={(e) =>
+                dispatch({ type: "RATING", payload: parseInt(e.target.value) })
+              }
+            />
+            <label htmlFor="filter-star-three">
+              {" "}
+              3 <i className="fas fa-star"></i> & Up
+            </label>
+          </div>
+          <div className="filter-content">
+            <input
+              type="radio"
+              name="filter-rating-star"
+              id="filter-star-two"
+              value="2"
+              checked={rating === 2}
+              onChange={(e) =>
+                dispatch({ type: "RATING", payload: parseInt(e.target.value) })
+              }
+            />
+            <label htmlFor="filter-star-two">
+              {" "}
+              2 <i className="fas fa-star"></i> & Up
+            </label>
+          </div>
+          <div className="filter-content">
+            <input
+              type="radio"
+              name="filter-rating-star"
+              id="filter-star-one"
+              value="1"
+              checked={rating === 1}
+              onChange={(e) =>
+                dispatch({ type: "RATING", payload: parseInt(e.target.value) })
+              }
+            />
+            <label htmlFor="filter-star-one">
+              {" "}
+              1 <i className="fas fa-star"></i> & Up
```

Comment:
All these are repetitive codes. Why don't you use a map function for rating?

Reply:
Thanks Ranita will refactor at the end!